src/Policies/KeywordTypePolicy.php

```diff
@@ -1,49 +0,0 @@
-<?php
```

Comment:
Removed this policy, I don't believe these are included in the migration anymore either

Reply:
That's correct. Thanks